fab/operations/release.py

```diff
@@ -281,3 +281,16 @@ def mark_keep_until(keep_days):
     until_date = (datetime.utcnow() + timedelta(days=keep_days)).strftime(DATE_FMT)
     with cd(env.code_root):
         sudo('touch {}{}'.format(KEEP_UNTIL_PREFIX, until_date))
+
+
+@roles(ROLES_ALL_SRC)
+@parallel
+def apply_patch(filepath):
+    destination = '/home/{}/patch.patch'.format(env.user)
```

Comment:
could do, my initial logic was i didn't want a bunch of files hanging around, but now thinking about it again it'd be useful to have the date in the filename so we know when it happened